spec/controllers/bulk_updates_controller_spec.rb

```diff
@@ -114,7 +114,63 @@
       it 'redirects the user to the manage metadata page' do
         sign_in
 
-        post :new, search_field: 'short_name', search_query: 'dunno'
+        post :new
+
+        expect(response).to redirect_to(manage_metadata_path)
```

Comment:
Maybe this should go to `new_bulk_updates_search_path` like the others? I don't think we want to use Manage Metadata as an entry point generally.